sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_evaluate.py

```diff
@@ -92,6 +92,41 @@ def evaluate(
         output_path=None,
         **kwargs
 ):
+    """Evaluates target or data with built-in evaluation metrics
+
+    :keyword evaluation_name: Display name of the evaluation.
+    :paramtype evaluation_name: Optional[str]
+    :keyword target: Target to be evaluated. `target` and `data` both cannot be None
+    :paramtype target: Optional[Callable]
+    :keyword data: Path to the data to be evaluated or passed to target if target it set.
+        Only .jsonl format files are supported.  `target` and `data` both cannot be None
+    :paramtype data: Optional[str]
+    :keyword task_type: Task type for evaluation. This helps to pick a set of pre-defined metrics.
+        Supported values are `qa` and `chat`
+    :paramtype task_type: str
+    :keyword metrics_list: List of metrics to calculate.A default list is picked based on task_type of not set.
+    :paramtype metrics_list: Optional[List[str]]
+    :keyword model_config: GPT configuration details needed for AI-assisted metrics.
+    :paramtype model_config: Dict[str, str]
+    :keyword data_mapping: GPT configuration details needed for AI-assisted metrics.
+    :paramtype data_mapping: Dict[str, str]
+    :keyword output_path: Save evaluation artifacts to local folder path
```

Comment:
Can we reword this? Would a better description be something like "The local folder path to save evaluation artifacts to"

Also, what happens if none is provided?

Reply:
It is optional. It will just be logged in the run given the AI client in Azure AI Studio and not saved to any local folder.

Reply:
"The local folder path to save evaluation artifacts to" sounds good